phpunit.xml.dist

```diff
@@ -16,6 +16,12 @@
 		<directory>tests/Common</directory>
 		<directory>tests/Options</directory>
 		<directory>tests/S3Uploader</directory>
+		<file>tests/File/CommonFilesTests.php</file>
+		<file>tests/File/AwsFileTest.php</file>
+	</testsuite>
+
+	<testsuite name="files-azure">
```

Comment:
~~Sem by se měl přihodit i `tests/File/CommonFilesTests.php`~~

Reply:
Pojmenoval bych možná radši `file-storage-azure` ať je to konzistentnější se zbytkem.